ONEARMY/community-platform
# feat: named research collaborators can access/edit research

PR Checklist

- [ ] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

Adds the ability for named research contributors to access a Research Article and any associated Research Article Updates.

Testing it out:

1. Add a username to the field; _Who have you been collaborating on this Research with?_
2. Log in with the username from step one.
3. View the Research article, you should be able to see the `Edit` button
4. Edit the description, existing Research Update or create a new Update.
5. Everybody can Research now 🔬 

There are 3 distinct parts to this PR:

1. Access control via the React based routing logic
2. Persisting collaborator information to Research Updates
3. Rendering the collaborator information on Research Updates

## Git Issues

Part of #2144
